pyproject.toml

```diff
@@ -15,6 +15,7 @@ dependencies = [
     "spacy>=3.0,<4.0",
     "word2number",
 #    "en-parsigs @ https://huggingface.co/royashcenazi/en_parsigs/resolve/main/en_parsigs-any-py3-none-any.whl",
+    "spellchecker<= 0.5.5",
```

Comment:
can you send the github link to this dependancy? 
We need to make sure there are no network (api calls) in order to keep patient data secured
other than that looks good!

Reply:
The link is in the description of the issue, I can send it in PR if you wish.

Reply:
@itay-goldraich seems the dependancy than should be `pyspellchecker` and not `spellchecker` no? 
Fix this, run tests and we can merge :)

Reply:
Fixed.